### DISCUSSION TITLE
Next page

### DISCUSSION THREAD

**Discussion by jcarlo2** [dolanmiu/docx#1932] [Q&A] [answered]
Is there a feature where I can go to the next page and start writing instead of the next line?
` const doc = new Document({
        sections: [
            {
                children: [
                    new Paragraph() --> This is for first page,
                    new Paragraph() --> This is for the second page
                ],
            },
        ],
    });`

**Comment by dolanmiu** [comment, marked as answer]
Yes!

It is called PageBreak:

https://docx.js.org/#/usage/paragraph?id=page-break
